### ISSUE TITLE
(urn:cts:greekLit:tlg5025.tlg002.1st1K-grc1) new edition

### ISSUE THREAD

**Issue by charhoughton** [OpenGreekAndLatin/First1KGreek#2460]
see editing card for volunteer names, XML, etc: https://trello.com/c/Rdb9IbSo/209-edit-poetaeminoresgra02gais-scholia-in-hesiodem
publishing card: https://trello.com/c/zcjvla0D/287-publish-poetaeminoresgra02gais

This text will require reformatting to add another level of citation for each scholion.

**Comment by charhoughton** [comment]
Yes, I know Bruce for a while was doing some work on files between the Lace export and XML work, so I held off on adding files. Here's the raw file from LACE:
[poetaeminoresgra02gais.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/7367146/poetaeminoresgra02gais.zip)

**Comment by lcerrato** [comment]
@charhoughton 
Thanks. I know there are some files I've skipped for that reason. Just working backwards. (Plenty of other stuff to do!)
